src/libAtomVM/interop.h

```diff
@@ -95,14 +96,32 @@ NO_DISCARD enum UnicodeConversionResult interop_chardata_to_bytes(term t, uint8_
  * @details Allows to quickly translate atoms to any integer constant. This function is useful for
  * creating switch statements for atom values.
  * A linear search is performed, so table entries should be sorted by frequency.
- * @param table an array AtomStringIntPair structs, teminated with a default entry marked with
+ * @param table an array AtomStringIntPair structs, terminated with a default entry marked with
  * SELECT_INT_DEFAULT macro.
  * @param atom the atom used for comparison.
  * @param global the global context.
  * @returns the found int value which corresponds to the given atom.
  */
 int interop_atom_term_select_int(const AtomStringIntPair *table, term atom, GlobalContext *global);
 
+/**
+ * @brief Finds the first matching atom in an atoms table .
+ *
+ * @details Allows to quickly translate integer constants to an atom in an atoms table.
+ * This function is the inverse of interop_atom_term_select_int.
+ * IMPORTANT: This function can be used in a switch statement, but it is more likely
+ * to be used to search for an atom based on a value, where the atom is used to return
+ * a term back to the user.  When using interop_atom_term_select_int, users should generally
+ * use a switch statement, for better readability.
```

Comment:
Sorry, I thought about my previous comment again, and I realized that it was already fine as it was, so this `IMPORTANT` note should be reverted.
Sorry for the wrong feedback.

Reply:
Removed "IMPORTANT:" note.